toolbox/ci2gubernator/lib/ci2gubernator.go

```diff
@@ -67,6 +68,10 @@ func NewConverter(bucket, org, repo, job string, build int) *Converter {
 		job:           job,
 		build:         build,
 	}
+	if presubmit {
```

Comment:
instead of a boolean, how about using a suffix string so it becomes reusable for other cases as well? And then you can just put everything in the same sprintf line, assuming the default suffix is just an empty string.

Reply:
Done